### DISCUSSION TITLE
【委托】

### DISCUSSION THREAD

**Discussion by MiaoJiSir** [share121/inter-knot#481] [其他]
![微信图片_20240810093722](https://github.com/user-attachments/assets/0c3774b1-c0b9-4503-985b-8f5a8d7a07d9)
[委托]物流事故！急求绳匠！
我订购的食材在运输中出了状况，「天马速运」的订单显示，货车一头闯进了十四分街的伴生空洞里，再也没有出来。
请绳匠帮我去看看货车的情况，食材拿不拿回来已经不重要了，只希望货车司机没事！

**Comment by MiaoJiSir** [comment]
这个委托我接了

**Comment by warning-Kirov-reporting** [comment]
我怎么记得我已经做过这个委托了

**Comment by KUNKUN4** [comment]
奖励呢？

**Comment by yrjmdqmmx** [comment]
hhhhhhhhhhhhhhhh

**Comment by xinxingjie** [comment]
我看这喵吉长官也是个尤物
